CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.2.2 (2020-08-31)
+
+- Fixed the overlay service to allow for overlays which cover the base element. [#199](https://github.com/blackbaud/skyux-core/pull/199)
```

Comment:
I would lean toward making this an enhancement, added support for the `verticalAlignment` setting when the `placement` is set to `'above'` or `'below'`.

Reply:
Updated